src/lib.rs

```diff
@@ -75,7 +76,19 @@ pub trait Field:
     const ONE: Self;
 
     /// Returns an element chosen uniformly at random using a user-provided RNG.
-    fn random<R: RngCore + ?Sized>(rng: &mut R) -> Self;
+    fn random<R: RngCore + ?Sized>(rng: &mut R) -> Self {
+        Self::try_from_rng(rng)
+            .map_err(|e: Infallible| e)
```

Comment:
I confirmed that due to a blanket `impl<R: RngCore> TryRngCore<Error = Infallible> for R`, this does not impose any additional constraints or require any changes to the trait method documentation.